src/plays/cricket-game/styles.css

```diff
@@ -0,0 +1,282 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700;800;900&family=Signika+Negative:wght@300;400;500;600;700&display=swap');
+
+/* enter styles here */
+
+/* The main body of game */
+.cricket-home-body {
+    background-image: url("./assets//cricket-ground.jpg");
+    box-shadow: 0px 0px 7px 2px rgba(0, 0, 0, 0.2), 
+    0px 0px 7px 2px rgba(0, 0, 0, 0.2),
+    0px 0px 7px 2px rgba(0, 0, 0, 0.2),
+    0px 0px 7px 2px rgba(0, 0, 0, 0.2);
+
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+}
+
+.cricket-home-body img {
+    pointer-events: none;
+    user-select: none;
+    cursor: none;
+}
+
+.cricket-home-body * {
```

Comment:
You do not need to give * here... just move the font style to the '.cricket-home-body' declaration

Reply:
@atapas I already tried putting 
```css
.cricket-home-body {
    font-family: "Signika Negative", Arial, Helvetica, sans-serif;
}
```
Earlier... It did not work, the font only applied to that particular class but the child elements didn't got that font.

Reply:
Kindly ignore my earlier comment.
I've resolved the issue in https://github.com/reactplay/react-play/pull/972/commits/d2ef717fad6d97d7a4129d67e1c79563bb4d5ef2